src/wireviz/Harness.py

```diff
@@ -277,12 +344,28 @@ def create_graph(self) -> Graph:
                     loop_dir = "e"
                 else:
                     raise Exception("No side for loops")
-                for loop in connector.loops:
+                for i in range(1, len(loopPins)):
```

Comment:
So this would allow loops with more than two pins?
I think there is a check for that somewhere in the Connector Dataclasses.

Reply:
Yes the check was disabled

Reply:
Removed the check corectly